src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs

```diff
@@ -130,6 +130,43 @@ public bool HasAttributeDerivedFrom(MemberInfo memberInfo, Type baseAttributeTyp
             return false;
         }
 
+        /// <summary>
+        /// Returns true when specified class/member has attribute which implements specific interface.
+        /// </summary>
+        /// <param name="memberInfo">The member info.</param>
+        /// <param name="interfaceType">The interface type.</param>
+        /// <param name="inherit">Should look at inheritance tree.</param>
+        /// <returns>An object derived from Attribute that corresponds to the instance of found attribute.</returns>
```

Comment:
Returns bool

Reply:
Cant we use HasAttributeDerivedFrom ? 
If not, let's rename the method to HasAttributeImplementingInterface/HasAttributeImplementing/ HasAttributeImplementedFrom

Reply:
Done.